stdlib/src/builtin/string.mojo

```diff
@@ -574,22 +574,28 @@ fn _is_ascii_lowercase(c: UInt8) -> Bool:
 
 fn isspace(c: UInt8) -> Bool:
```

Comment:
I thought you already made this hidden, maybe something went wrong with a merge/rebase?

Reply:
Since it is only for a byte I thought of leaving it as is, tried putting it into String as a static method but the tests started failing.